.erb/configs/webpack.config.base.ts

```diff
@@ -48,4 +48,4 @@ export default {
       NODE_ENV: 'production',
     }),
   ],
-};
+} as Configuration;
```

Comment:
I think it would be cleaner to use a typed variable instead of a type cast.

e.g.:
```ts
const configuration: webpack.Configuration = {
......
}

export default configuration;

```
I have implemented a version with this variant in pull request #3072